simulated.datasets.py

```diff
@@ -1,7 +1,8 @@
-import matplotlib.pyplot as plt
+from   matplotlib import pyplot as plt
 import numpy as np
 import os
-import seaborn as sns; sns.set_style("white")
+import seaborn as sns; 
```

Comment:
Remove `;` now that set_style is on another line.

Reply:
Done